perma_web/static/js/folder-tree.module.js

```diff
@@ -103,23 +103,25 @@ FolderTreeModule.editNodeName = function (node) {
   }, 0);
 }
 
-FolderTreeModule.setSelectedFolder = function (node) {
-  var folderPath = this.folderTree.get_path(node),
-    data = "null";
-
+FolderTreeModule.getNodeData = function (node) {
   if (node.data) {
-    var folderId = node.data.folder_id;
-    var orgId = node.data.organization_id;
-    data = JSON.stringify({path: folderPath, orgId:orgId, folderId:folderId});
+    var data = {};
+    data.folderId = node.data.folder_id;
+    data.orgId = node.data.organization_id;
+    data.path = folderPath = this.folderTree.get_path(node);
   }
+  return data;
+}
```

Comment:
I'm confused what this function returns if `node.data` is false. Maybe `var data = {}` wants to be outside the `if`?